frontend/copr-frontend.spec

```diff
@@ -296,17 +296,13 @@ EOF
 %py_byte_compile %{__python3} %{buildroot}%{_datadir}/copr/coprs_frontend/coprs
 %py_byte_compile %{__python3} %{buildroot}%{_datadir}/copr/coprs_frontend/alembic
 
+install -m0644 -D conf/copr-frontend.sysusers.conf %{buildroot}%{_sysusersdir}/copr-frontend.conf
+
 %check
 %if %{with check}
 ./run_tests.sh -vv --no-cov
 %endif
 
-%pre
```

Comment:
can you please explain why to remove the entire `%pre`?
that means downstream without `rpm > 4.19` should create user manually

Reply:
That's right.  Fedora 41+ is on RPM 4.20+, so that is non-issue for us - and since you pre-create usernames in kube deployments it should be no problem for openEuler Copr, too.